Cour-de-cassation/label
# Keep label treatments when reimport

## Issue description :
Allow possibility to reimport labelTreatments when reimporting a specific decision

## Describe your changes :
- delete strange usage of labelTreatmentType
- refacto of preAssignation
- map old labelTreatments to a new labelTreatment
- at export, send only new treatments and reorder before inserting in sder database (in sder repo)

## How to test :
Better test in dev or preprod env, import a specific decision with variation (keepLabelTreatments, with preAssignation, etc.) and check decision at export

## Checklist before requesting a review
> Lors du retraitement d'une décision nous avons désormais la possibilité de conserver les traitements déjà appliqués sur cette décision.
